tools/scripts/ros_setup/pkg_dependencies/getRos2.sh

```diff
@@ -0,0 +1,20 @@
+#!bin/bash
+
+echo "#########################################################"
+echo "#  prepration machine to install Vision Package         #"
+echo "#            AURTHOR\ ELabbasSalah                      #" 
+echo "#########################################################"
```

Comment:
- Prepare machine to install VAUV packages, I think preparation machine is not correct
- Please remove your name from the script, this is not totally professorial , you have already been added to the contributes and the commit is under your name, so all your rights are reserved no worries